boilerplates/fe-angular/README.md

```diff
@@ -0,0 +1,82 @@
+# fe-angular
+
+Angular quickstarter project
+
+## Purpose of this quickstarter
+ 
+ This quick starter gives you a new Angular project. This project contains basic setup for docker, jenkins, sonarqube and OpenShift. 
+
+## What files / architecture is generated?
+ ```
+.
+├── Jenkinsfile
+├── README.md
+├── angular.json
+├── docker
+│   ├── Dockerfile
+│   └── nginx.vh.default.conf.nginx
+├── e2e
+│   ├── protractor.conf.js
+│   ├── src
+│   │   ├── app.e2e-spec.ts
+│   │   └── app.po.ts
+│   └── tsconfig.e2e.json
+├── package.json
+├── sonar-project.properties
```

Comment:
same here # this is the file that controls the sonarqube scan plugin

Reply:
added